.github/workflows/ci.yml

```diff
@@ -142,57 +142,12 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-${{ matrix.node-version }}-angular-${{ matrix.angular-version }}-yarn-
       - run: yarn install
-      - run: yarn delete-path-alias @<organization-hyphen>/<name-hyphen>
```

Comment:
Try removing everything except `11.0.x, `11.1.x` above.
```yaml
        angular-version: [11.0.x, 11.1.x]
```

Also do this for the `lib` job.